packages/react-ui-core/src/Counter/__tests__/Counter-test.js

```diff
@@ -44,9 +46,10 @@ describe('Counter', () => {
           count={1}
         />,
       )
-      wrapper.find('span[role="incrementUnit"]').first().simulate('click')
-      expect(wrapper.state('count')).toEqual(1)
+      wrapper.find('span[id="increment"]').first().simulate('click')
```

Comment:
How about you just grab the first child instead of doing this?  If you want the parent / top level component then pass an `id` there, but it should _not_ be defined within the component itself.  If you need me to explain, please ask.